tests/models/sam/test_processor_sam.py

```diff
@@ -152,7 +164,9 @@ def test_post_process_masks(self):
 
 @require_vision
 @require_tf
-class TFSamProcessorTest(unittest.TestCase):
+class TFSamProcessorTest(ProcessorTesterMixin, unittest.TestCase):
```

Comment:
I'd say you only need `ProcessorTesterMixin` for `SamProcessorTest`, so you can remove this here and the skipped tests as well

Reply:
Makes sense. However, I had to then add back the `prepare_image_inputs` methods. With this, I updated the comment.